src/umpire/resource/DefaultMemoryResource.hpp

```diff
@@ -23,8 +23,7 @@ namespace resource {
    */
 template <typename _allocator>
 class DefaultMemoryResource :
-  public MemoryResource,
-  private umpire::strategy::mixins::Inspector
+  public MemoryResource
```

Comment:
`MemoryResources` no longer track their own allocations, this is handled by an `AllocationTracker`.